docs/deploy/index.md

```diff
@@ -1,15 +1,18 @@
-# Deploying Browsertrix
+# Self-Hosting
+
+!!! info "Already signed up for Browsertrix?"
+
+    This guide is for developers and users who are self-hosting Browsertrix. If you've registered through [browsertrix.com](https://browsertrix.com/), you may be looking for the [user guide](../user-guide/index.md).
 
 Browsertrix is designed to be a cloud-native application running in Kubernetes.
 
-However, despite the name, it is perfectly reasonable (and easy!) to deploy Browsertrix locally using one of the many available local Kubernetes options.
+However, despite the name, it is perfectly reasonable to deploy Browsertrix locally using one of the many available local Kubernetes options.
```

Comment:
Since we've switched from Browsertrix Cloud to Browsertrix, maybe we remove the little clause about the name:

```suggestion
However, it is perfectly reasonable to deploy Browsertrix locally using one of the many available local Kubernetes options.
```

Reply:
Heh missed that!  +1